ui/app/hooks/useRetryTransaction.js

```diff
@@ -19,7 +19,8 @@ import { useMetricEvent } from './useMetricEvent'
  */
 export function useRetryTransaction (transactionGroup) {
   const { primaryTransaction, initialTransaction } = transactionGroup
-  const gasPrice = primaryTransaction.txParams.gasPrice
+  // Signature requests do not have a txParams, but this h
```

Comment:
I like this comment, but it's a bit of a cliff-hanger.

Reply:
what... in the hell